source/includes/swift/_containers.md

```diff
@@ -0,0 +1,196 @@
+      "displaySize": {
+        "value": "0",
+        "unitKey": "units.filesizes.B"
+      },
+      "size": 0,
+      "objectCount": 0,
+      "accessType": "private",
+      "locked": false,
+      "storagePolicy": "Quebec"
+    },
+    {
+      "id": "public",
+      "name": "public",
+      "isPublic": true,
+      "displaySize": {
+        "value": "81",
+        "unitKey": "units.filesizes.B"
+      },
+      "size": 81,
+      "objectCount": 6,
+      "accessType": "public",
+      "locked": false,
+      "storagePolicy": "Quebec"
+    }
+  ],
+  "metadata": {
+    "recordCount": 2
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/buckets</code>
+
+Retrieve a list of all containers in a given [environment](#administration-environments)
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/>*string* | The id of the container 
+`name` <br/>*string* | The name of the container
+`isPublic`<br/>*boolean* | Specifies if the container is public or not
+`displaySize`<br/>*Object* | Object containing the information to display the size in the UI.
+`size`<br/>*string* | The size in bytes of the container
+`objectCount`<br/>*string* | The number of object which are part of the container
+`accessType`<br/>*string* | The access type of the container. It can either be public or private
+`locked`<br/>*string* | Identify if the container is locked or not.
+`storePolicy`<br/>*string* | The store policy applying to the container.
+
+
+<!-------------------- RETRIEVE A CONTAINER -------------------->
+
+#### Retrieve a container
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/swift/example/buckets/private"
+
+# The above command returns JSON structured like this:
+```
+
+```json
+{
+  "id": "private",
+  "name": "private",
+  "isPublic": false,
+  "displaySize": {
+    "value": "0",
+    "unitKey": "units.filesizes.B"
+  },
+  "size": 0,
+  "objectCount": 0,
+  "accessType": "private",
+  "locked": false,
+  "storagePolicy": "Quebec"
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/buckets/:id</code>
+
+Retrieve a container in a given [environment](#administration-environments)
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/>*string* | The id of the container 
+`name` <br/>*string* | The name of the container
+`isPublic`<br/>*boolean* | Specifies if the container is public or not
+`displaySize`<br/>*Object* | Object containing the information to display the size in the UI.
+`size`<br/>*string* | The size in bytes of the container
+`objectCount`<br/>*string* | The number of object which are part of the container
+`accessType`<br/>*string* | The access type of the container. It can either be public or private
+`locked`<br/>*string* | Identify if the container is locked or not.
```

Comment:
This is a little confusing, is this supposed to be a boolean as in the above example?
if it is, maybe something like "Specifies if the container is locked or not"

Reply:
forgot to change it. will do

Reply:
f47db080986d4eed0065f2b048c24c5faeafb7be